backend/test.env

```diff
@@ -0,0 +1,5 @@
+TOKEN_SECRET=TEST123456
+NEO4J_URI=bolt://18.204.8.8:33335
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=twos-pages-diagnosis
+NEO4J_DATABASE=
```

Comment:
test Variables

Reply:
⭐ For requesting a review and reviewing another team's PR.
You asked review from us.. and we are sure that you will review us as soon as you have our request. so this star will be yours for sure...